_vcsh_bash

```diff
@@ -128,11 +132,17 @@ _vcsh () {
 
 	# git command on repository
 	if [[ -n "${repos[$cmd]}" ]]; then
-		_vcsh_git_command $subcword
+		local REPO_DIR="${XDG_CONFIG_HOME:-$HOME/.config}/vcsh/repo.d/${cmd}.git"
```

Comment:
Shouldn't this be using `$VCSH_REPO_D` or something similar instead of rolling your own guess as to the repository location?

Reply:
I didn't test it, but I'm pretty sure that's what the variable *is for*.

Reply:
Using `$VCSH_REPO_D` will be more robust than the proposed line.

Reply:
`VCSH_REPO_DIR` is [defined in _vcsh](https://github.com/RichiH/vcsh/blob/main/_vcsh#L101-L102) also since the `vcsh` script is not active for command completion. So really it is not much different than what I have other than the name of the local variable. I will rename it to be consistent with the other locations it is defined.

Reply:
True, but it's defined in such a way that it's only set **if** it doesn't already exist. In the case of being inside `vcsh enter` it will be set in the host environment and that definition should be a NOOP. Using that variable is still the right think to do.